USERS.md

```diff
@@ -117,6 +117,11 @@ Users (Alphabetically)
       U: Networking, network policy, and network visibility
       L: https://cloud.google.com/blog/products/containers-kubernetes/bringing-ebpf-and-cilium-to-google-kubernetes-engine
 
+    * N: JUMO
+      D: JUMO is using Cilium as their CNI plugin for all of their AWS-hosted EKS clusters
+      U: Networking, network policy, network visibility, cluster mesh
+      Q: @Matthieu ANTOINE, @Carlos Castro, @Joao Coutinho
```

Comment:
This is usually a list of GitHub or Twitter handles. Alternatively I think it's also fine to list the full names but you'll have to drop the `@`s.

Reply:
Oh my bad. These are supposed to be Cilium Slack Nicks, but maybe I I could put them in a clearer way.

Reply:
Oh, I always forget Slack allows for white spaces :facepalm: Maybe just adding `(Slack)` at the end of the line would be enough?

Reply:
Ah sorry 😅  For sure will do that!

Reply:
Done!

Reply:
Thanks!